src/providers/contexts.js

```diff
@@ -1,12 +1,27 @@
 // @flow
 import { createRealmContext } from "@realm/react";
 import { createContext } from "react";
+import RNFS from "react-native-fs";
 // eslint-disable-next-line import/extensions
 import realmConfig from "realmModels/index";
 
+import { log } from "../../react-native-logs.config";
+
+const logger = log.extend( "contexts.js" );
+
+async function checkOnRealmPath( context ) {
+  const existResult = await RNFS.exists( realmConfig.path );
+  logger.info( `${context} does ${realmConfig.path} exist? ${existResult}` );
+}
+checkOnRealmPath( "before creating realm context" );
+
 const ExploreContext: Object = createContext<Function>( );
 const ObsEditContext: Object = createContext<Function>( );
 const RealmContext: Object = createRealmContext( realmConfig );
+checkOnRealmPath( "after creating realm context" );
+setTimeout( ( ) => {
+  checkOnRealmPath( "5s after creating realm context" );
```

Comment:
This is some paranoid logging on my part. I don't think `createRealmContext()` creates the Realm file specified in the config synchronously, so I made this timeout to really convince myself it was getting created. I can remove it if it's annoying, but I wanted to leave it in for a little while in case this commit doesn't actually fix the bug.